docker.go

```diff
@@ -440,12 +440,15 @@ func (c *DockerContainer) Exec(ctx context.Context, cmd []string) (int, io.Reade
 		Detach:       false,
 		AttachStdout: true,
 		AttachStderr: true,
+		Tty:          false,
 	})
 	if err != nil {
 		return 0, nil, err
 	}
 
-	hijack, err := cli.ContainerExecAttach(ctx, response.ID, types.ExecStartCheck{})
+	hijack, err := cli.ContainerExecAttach(ctx, response.ID, types.ExecStartCheck{
+		Tty: true,
```

Comment:
This `true` value does not mean that it will use Tty **but** to check if there is Tty 🤷

Reply:
I am not sure if I understood the concept correct, but for .NET `tty` is always `false`. There is no interaction, we run the command and return the result.

Reply:
Indeed: when we create the command, it's always `false` https://github.com/testcontainers/testcontainers-go/pull/624/files#diff-b68159b6168f099a639e8c6f96d75f23333ab0f0505ee22a98268748db17ae4cR443

What I was not aware of, is the ExecAttach part